api/v1alpha1/onload_types.go

```diff
@@ -59,6 +59,13 @@ type DevicePluginSpec struct {
 	// ImagePullPolicy is the policy used when pulling images.
 	// More info: https://kubernetes.io/docs/concepts/containers/images#updating-images
 	ImagePullPolicy v1.PullPolicy `json:"imagePullPolicy,omitempty"`
+
+	// +optional
+	// MaxPodsPodsPerNode is the number of kubernetes devices that the device
+	// plugin should register to the kubelet. Notionally this is equivalent to
+	// the number of pods that can request an onload resource on each node.
+	// If omitted this will default to 100
```

Comment:
This change updates the base crd in `config/samples/crd/bases/onload.amd.com_onloads.yaml` to specify the defaults.
Additionally it will fill in these values for any onload CR that is created.
I think that that is slightly nicer behaviour as a user can determine what value it was set to by running `oc get onload ...`